Software/vortex_ws/src/vision/README.md

```diff
@@ -1,15 +1,15 @@
 # Vision
 
 ## Description
-Vision is a ros2 package which responsible for mission recognition and object detection. 
-* The vision module passes the data to the mission planning module.
-* Data contain logs from the deep learning model and image analysis model.
-* The logs from the deep learning models contain the detected object class ,and it's coordinates
+Vision is a ros2 package of the AUV vision module, which responsible for mission recognition and object detection. 
```

Comment:
Please try to write more descriptive description 
For example:
The Vision package is responsible for  Image processing, image analysis and performing multiple deep learning algorithms on the video streams coming from the AUV  cameras' to detect it's targets